doc/nrf/protocols/matter/getting_started/transmission_power.rst

```diff
@@ -0,0 +1,154 @@
+|                                                              | nrf5340dk_nrf5340        | 0                                                    |
+|                                                              +--------------------------+------------------------------------------------------+
+|                                                              | nrf21540dk_nrf52840      | 0                                                    |
++--------------------------------------------------------------+--------------------------+------------------------------------------------------+
+| :ref:`Template (MTD) <matter_template_sample>`               | nrf52840dk_nrf52840      | 8                                                    |
+|                                                              +--------------------------+------------------------------------------------------+
+|                                                              | nrf5340dk_nrf5340        | 3                                                    |
+|                                                              +--------------------------+------------------------------------------------------+
+|                                                              | nrf21540dk_nrf52840      | 20                                                   |
++--------------------------------------------------------------+--------------------------+------------------------------------------------------+
+| :ref:`Widow Covering (SSED) <matter_window_covering_sample>` | nrf52840dk_nrf52840      | 0                                                    |
+|                                                              +--------------------------+------------------------------------------------------+
+|                                                              | nrf5340dk_nrf5340        | 0                                                    |
+|                                                              +--------------------------+------------------------------------------------------+
+|                                                              | nrf21540dk_nrf52840      | 0                                                    |
++--------------------------------------------------------------+--------------------------+------------------------------------------------------+
+
+Changing TX power for Thread
+****************************
+
+To change the radio TX power used by the Thread protocol, set the :kconfig:option:`CONFIG_OPENTHREAD_DEFAULT_TX_POWER` to the desired dBm value in a range from ``-40`` to ``20`` dBm.
+
+The following table lists the maximum output power values in dBm for each board.
+The maximum value of 20 dBm is only recommended for devices that are using :ref:`radio Front-End Modules <ug_radio_fem>`.
+
++--------------------------+--------------------------------+
```

Comment:
We can add information about the maximum power of BLE and WIFI described in a comment above to this table.

https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf7002%2Fchapters%2Felspec%2Fdoc%2Felectrical_specification.html, Table 3

For nrf52840dk_52840:

max BLE power is +8 dBm -> CONFIG_BT_CTLR_TX_PWR_PLUS_8
min BLE power is -40 dBm -> CONFIG_BT_CTLR_TX_PWR_MINUS_40
For nrf5340dk_nrf5340 & nrf7002dk_nrf5340:

max BLE power is +3 dBm -> CONFIG_BT_CTLR_TX_PWR_PLUS_3
min BLE power is -40 dBm -> CONFIG_BT_CTLR_TX_PWR_MINUS_40